src/ILCompiler.ReadyToRun/src/Compiler/ReadyToRunCodegenCompilation.cs

```diff
@@ -70,6 +78,11 @@ protected override void ComputeDependencyNodeDependencies(List<DependencyNodeCor
                     continue;
 
                 MethodDesc method = methodCodeNodeNeedingCode.Method;
+                if (!NodeFactory.CompilationModuleGroup.ContainsMethodBody(method, unboxingStub: false))
```

Comment:
We should just make a new node for these, similar to how we make `ExternMethodSymbolNode` on the RyuJIT side for these in the node factory (and same for UTC codegen, etc.). It fits the model better. There will be places where we'll want to operate on just the compiled method bodies and having a `MethodCodeNode` that is not actually a compiled method body will make things more awkward there.

Reply:
This shouldn't be needed, in my final changes I do have a different object for the imported methods. I'll remove this.